### PR TITLE
Update dependencies to allow this sample to work

### LINKED ISSUE #5: Sample require update
The management library is outdated (0.0.1-preview, where current version is 0.2.0-preview).
The following exception is thrown:

```
Microsoft.Rest.Azure.CloudException: 'The api-version '2015-08-01' is invalid. The supported versions are '2017-05-10,2017-05-01,2017-03-01,2016-09-01,2016-07-01,2016-06-01,2016-02-01,2015-11-01,2015-01-01,2014-04-01-preview,2014-04-01,2014-01-01,2013-03-01,2014-02-26,2014-04'.'
```

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #5

### CODE REVIEW

**Comment by SeanFeldman** [comment]
Microsoft.Azure.Management.ServiceBus [1.0.0 was just released](https://www.nuget.org/packages/Microsoft.Azure.Management.ServiceBus/1.0.0). Will update PR.

**Comment by SeanFeldman** [comment]
@Azure/azure-service-bus-write please review.

**Comment by vinaysurya** [comment]
> using Microsoft.Azure.Management.ServiceBus.Models;

nit: move this inside the namespace

---
Refers to: src/service-bus-dotnet-management/AppOptions.cs:1 in d4aa9be. [](commit_id = d4aa9be74e282f5acea96c7578b399f685334158, deletion_comment = False)

File: src/service-bus-dotnet-management/AppOptions.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Azure.Management.ServiceBus.Models;
+
```
Comment (vinaysurya) [inline src/service-bus-dotnet-management/AppOptions.cs]: Add missing copyright statement

File: src/service-bus-dotnet-management/AppOptions.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+namespace service_bus_dotnet_management
```
Comment (vinaysurya) [inline src/service-bus-dotnet-management/AppOptions.cs]: may be azure_service_bus* ?

Comment (SeanFeldman) [inline src/service-bus-dotnet-management/AppOptions.cs]: This was the original namespace for the sample. I didn't touch it.
Will rename namespace to `AzureServiceBusDotNetManagement`

**Comment by SeanFeldman** [comment]
@vinaysurya incorporated your review.

image

**Comment by vinaysurya** [comment]
Thanks for these changes @SeanFeldman!  Apologies for the delay, TBH, we weren't aware this repo already existed... Only looked at it today when a User in the servicebus-dotnet community pointed out at this repo and asked about a newer version.

**Comment by SeanFeldman** [comment]
That is awesome. Good thing an issue was raise 😃
Anyway, I'm done with changes.

**Pull request merged**